packages/cardano-services-client/scripts/createVersionSource.js

```diff
@@ -1,14 +1,10 @@
 const fs = require('fs');
 const path = require('path');
 
-const packageJson = require('../package.json');
 const openApiSpec = require('../../cardano-services/src/Http/openApi.json');
 
-const contents = `// auto-generated using ${__filename}
-export const version = {
-  api: '${openApiSpec.info.version}',
-  software: '${packageJson.version}'
-};
+const contents = `// auto-generated using ../scripts/createVersionSource.js
+export const apiVersion = '${openApiSpec.info.version}';
```

Comment:
Note this will likely change to use individual provider versions, as this is currently just using the top level HTTP server schema, but it's out of scope for this PR.